.github/workflows/ci.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Select Xcode version
       run: sudo xcode-select -switch /Applications/Xcode_11.4.app
-    - name: Build project
-      run: swift build
+    - name: Build project (Release)
+      run: swift build -c release
     - name: Run tests
       run: swift test
```

Comment:
maybe add a comment that this still tests in Debug?

That said, in practice I've found that getting tests working in Release can be more helpful than running in Debug.

Reply:
As far as I understand you can't use `@testable` in release, that may be completely wrong though.

Reply:
I use a release configuration in CacheAdvance [build script](https://github.com/dfed/CacheAdvance/blob/b0f588d8541258bd87d11fae3cdf76a740da1311/Scripts/build.swift#L117) that [runs tests in CI](https://github.com/dfed/CacheAdvance/blob/master/Scripts/ci.sh#L8) driven by [TravisCI](https://github.com/dfed/CacheAdvance/blob/master/.travis.yml#L10), but I also am using `xcodebuild`. Not sure how using a release configuration would work using `swift`.